spreadsheets.qmd

```diff
@@ -342,12 +342,12 @@ bake_sale <- tibble(
 bake_sale
 ```
 
-You can write data back to disk as an Excel file using the `write_xlsx()` from the [writexl package](https://docs.ropensci.org/writexl/):
+You can write data back to disk as an Excel file using the `write_xlsx()` function from the [writexl package](https://docs.ropensci.org/writexl/):
 
 ```{r}
 #| eval: false
 
-write_xlsx(bake_sale, path = "data/bake-sale.xlsx")
+write_xlsx(bake_sale, path = "data/bake_sale.xlsx")
```

Comment:
This matches the name in the caption text for the associated `Figure 20.4: Spreadsheet called bake_sale.xlsx in Excel.`: `bake_sale` with an underscore.